requirements/lint

```diff
@@ -0,0 +1,4 @@
+flake8
+flake8-isort
```

Comment:
I think `isort` will have give troubles without a `-r base` here

Reply:
troubles how so? It works in practice.

Reply:
Yes, I just tried it and I can confirm. It used to classify as first or third party differently depending on whether the library was available or not in the current environment, but it seems that it's no longer the case.